.github/ISSUE_TEMPLATE/2.bug.md

```diff
@@ -0,0 +1,68 @@
+---
+name: Bug report 🐛
+about: Create a bug report for Material-UI.
+---
+
+<!-- Provide a general summary of the issue in the Title above -->
+
+<!--
+  Thank you very much for contributing to Material-UI by creating an issue! ❤️
+  To avoid duplicate issues we ask you to check off the following list.
+-->
+
+<!-- Checked checkbox should look like this: [x] -->
+
+- [ ] This is not a v0.x issue. <!-- (v0.x is no longer maintained) -->
+- [ ] I have searched the [issues](https://github.com/mui-org/material-ui/issues) of this repository and believe that this is not a duplicate.
+
+## Current Behavior 😯
+
+<!-- Describe what happens instead of the expected behavior. -->
+
+## Expected Behavior 🤔
+
+<!-- Describe what should happen. -->
+
+## Steps to Reproduce 🕹
+
+<!--
+  Provide a link to a live example (you can use codesandbox.io) and an unambiguous set of steps to reproduce this bug.
+  Include code to reproduce, if relevant (which it most likely is).
+
+  This codesandbox.io template _may_ be a good starting point:
+  https://codesandbox.io/s/github/mui-org/material-ui/tree/master/examples/create-react-app
+
+  If you're using typescript a better starting point would be
+  https://codesandbox.io/s/github/mui-org/material-ui/tree/master/examples/create-react-app-with-typescript
+
+  If YOU DO NOT take time to provide a codesandbox.io reproduction, should the COMMUNITY take time to help you?
+-->
+
+Steps:
+
+1.
```

Comment:
No more increment?

Reply:
I hope the autoincrement will kick-in.

Reply:
It will on submit, but some users may not be aware of this feature when editing the file.

Reply:
Ok, will revert